@shared/api/internal.ts

```diff
@@ -412,6 +413,34 @@ export const retryAssetIcon = async ({
   return newAssetIcons;
 };
 
+export const getAssetDomains = async ({
```

Comment:
I looked into combining this with `getIconUrlFromIssuer`, I think it's cleaner separate rather than trying to squeeze them into the same method. My worry initially was performance, but I don't notice when making the call twice, and especially since they both using caching anyways